libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -96,12 +96,14 @@ const AP_Param::GroupInfo AC_WPNav::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("BRK_ACCEL",   9, AC_WPNav, _loiter_brake_accel_cmss, WPNAV_LOITER_BRAKE_ACCEL),
 
+#if AP_TERRAIN_AVAILABLE && (RANGEFINDER_ENABLED == ENABLED)
```

Comment:
This doesn't make sense here. There is no RANGEFINDER_ENABLED macro in this library or any header it includes.